src/plugins/bubble/_base.scss

```diff
@@ -0,0 +1,145 @@
+
+/*
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * Bubble wizard - Provisional CSS
+ *
+ * ++++++++
+ * This is a Provisional CSS, once stabilized it's include should be done from the theme.scss instead of _provisional.scss
+ * ++++++++
+ *
+ */
+/* Animations */
+
+@keyframes slideInFromRight {
+	0% {
+		transform: scale(0, 1); }
+	95% {
+		transform: scale(0, 1); }
+	100% {
+		transform: scale(1, 1); } }
+
+@keyframes pulseIn {
+	0% {
+		transform: scale(1, 1); }
+	15% {
+		transform: scale(1.15, 1.15); }
+	30% {
+		transform: scale(1, 1); }
+	65% {
+		transform: scale(1.3, 1.3); }
+	100% {
+		transform: scale(1, 1); } }
+
+@keyframes grow {
+	to {
+		transform: translateX(-50%) scale(0); } }
+
+.trans-left {
+	animation-delay: 0s;
+	animation-duration: 5s;
+	animation-iteration-count: 1;
+	animation-name: slideInFromRight;
+	animation-timing-function: ease-out;
+	transform-origin: 100% 50%;
+	will-change: scroll-position; }
+
+.trans-pulse {
+	animation: .5s linear 3.5s 1 pulseIn, .5s linear 15s 1 pulseIn, .5s linear 30s 1 pulseIn;
+	will-change: transform; }
+
+.loader-typing {
+	bottom: 30%;
+	height: 6px;
+	left: 30px;
+	position: absolute;
+	transform: translateX(-50%) translateY(-50%);
+	width: 26px; }
```

Comment:
No need for this css here (.loader-typing). This belongs to the chat itself. Please remove it.